script/setup.js

```diff
@@ -44,10 +43,11 @@ try {
 			gitRemoteFetch = await $`git remote -v | grep fetch`;
 		} catch {
 			console.log(
-				chalk.gray(
+				chalk.red(
 					"Could not populate default owner and repository. Did not detect a Git repository with an origin. "
 				)
```

Comment:
[Nit] I'm not sure this needs to be `.red`. It's not necessarily an error case. Users might use the template offline or in a non-Git directory (e.g. just making a quick local package before pushing somewhere). Thoughts?

Reply:
Right! Right! I planned this to be yellow, somehow missed it. Since it has the fallback.

Reply:
I don't think it should be yellow though 🤔 since it's not always a problem, I don't want to alarm folks. Thoughts?

Reply:
Right, let's go with grey then.

Reply:
Done, changed them.